.vscode/launch.json

```diff
@@ -0,0 +1,20 @@
+dd{
```

Comment:
Great putting .vscode in .gitignore Iñaki! But now you need to delete manually the tracked file. I think doing `git rm --cached .vscode` should work 👍

Reply:
not sure, what that is important. I tried to do it but it would't work, so I just deleted the folder manually. Should be excluded now